centers/tests/models/test_center.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+
+# Model
+from centers.models.center import Center
+
+# Test
+from centers.tests.models.test_region import RegionAllLayerTest
+
+# Util
+from centers.utils.center_hashids import get_encoded_center_hashid
+from centers.utils.center_hashids import get_center_hashids_object
+
+
+class CenterTest(RegionAllLayerTest, TestCase):
+
+    def setUp(self):
+        self.hashids = get_center_hashids_object()
+        super(CenterTest, self).setUp()
```

Comment:
http://makina-corpus.com/blog/metier/2014/python-tutorial-understanding-python-mro-class-search-path
you need to read MRO search path